.npmignore

```diff
@@ -0,0 +1,2 @@
+# Prevent publishing source code to NPM
+lib
```

Comment:
Use package.json's `files`, please - https://medium.com/@jdxcode/for-the-love-of-god-dont-use-npmignore-f93c08909d8d